lib/brakeman/report/report_markdown.rb

```diff
@@ -27,7 +27,7 @@ def initialize *args
   end
 
   def generate_report
-    out = "# BRAKEMAN REPORT\n\n" <<
+    out = +"# BRAKEMAN REPORT\n\n" <<
```

Comment:
Personally I really don't like `String#<<`... I wound up changing a lot of my old code from this to `Array#<<` + `#join`. It led to similarly sized diffs, array mutation (which I don't mind) instead of string mutation (which I do), and fits my current style a lot more. 

Not recommending this change here... just something to consider.

Reply:
Indeed... I'm sure there are a lot of improvements that could be made 😄